crates/da-clients/ethereum/Cargo.toml

```diff
@@ -15,14 +15,14 @@ alloy = { git = "https://github.com/alloy-rs/alloy", rev = "68952c0", features =
   "signer-wallet",
 ] }
 async-trait = { workspace = true }
-c-kzg = "1.0.0"
+c-kzg = { workspace = true }
 color-eyre = { workspace = true }
 da-client-interface = { workspace = true }
-dotenv = "0.15"
-mockall = "0.12.1"
-reqwest = { version = "0.12.3" }
+dotenvy.workspace = true
```

Comment:
Nitpick: should we change everything to one format? either `{ workspace = true }` or `.workspace = true`?